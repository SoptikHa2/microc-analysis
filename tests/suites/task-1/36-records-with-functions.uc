 inc(s) {
   (*s).c = (*s).c + 1;
   return *s;
 }

 new() {
   return { c: 0, inc: inc };
 }

 main() {
   var c, p;
   c = new();
   p = &c;
   return ((((c.inc)(p)).inc)(p)).c;
 }
