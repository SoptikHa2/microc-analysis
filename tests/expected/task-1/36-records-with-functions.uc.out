---
test: run suites/task-1/36-records-with-functions.uc
---

exit: 2
